### PR TITLE
Feat/popover slots rework

### PR DESCRIPTION
**Pull request by rustem-nasyrov**
Close: #1649

## Description
Added slots to the popover component, examples of its usage. Added translations. Refactored component.

image

Refactored and beautified vue-book demo example:
image

## Types of changes
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

**Review by m0ksem — APPROVED** [review]

File: packages/docs/src/locales/en/en.json [hunk 1]
```diff
@@ -1135,7 +1135,11 @@
       },
       "events": {},
       "methods": {},
-      "slots": {}
+      "slots": {
+        "icon": "By default replaces with a `va-icon` and icon name from `icon` property",
+        "title": "Custom title slot",
+        "body": " Custom body slot"
```
Comment (RVitaly1978) [inline packages/docs/src/locales/en/en.json:1141]:
```suggestion
        "body": "Custom body slot"
```

File: packages/docs/src/locales/ru/ru.json [hunk 2]
```diff
@@ -1122,7 +1122,11 @@
       },
       "events": {},
       "methods": {},
-      "slots": {}
+      "slots": {
+        "icon": "По умолчанию используется `va-icon` с названием из свойства компонента `icon`",
+        "title": "Слот для заголовка",
+        "body": " Слот для контента компонента"
```
Comment (RVitaly1978) [inline packages/docs/src/locales/ru/ru.json:1128]:
```suggestion
        "body": "Слот для контента компонента"
```

File: packages/docs/src/page-configs/ui-elements/popover/examples/Slots.vue [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+<script>
+import { VaButton, VaPopover } from 'vuestic-ui/src/components'
+
+export default {
+  name: 'Slots',
+
+  components: { VaButton, VaPopover },
+}
+</script>
```
Comment (RVitaly1978) [inline packages/docs/src/page-configs/ui-elements/popover/examples/Slots.vue:28]:
```suggestion
```

**Review by aluarius — APPROVED** [review]

File: packages/ui/src/components/va-popover/VaPopover.demo.vue [hunk 4]
```diff
@@ -1,39 +1,41 @@
...
             >
               <va-button>Top</va-button>
             </va-popover>
             <va-popover
-              placement="right"
+              placement="left"
               message="Placement Right"
+              class="mr-4"
             >
               <va-button>Right</va-button>
```
Comment (RVitaly1978) [inline packages/ui/src/components/va-popover/VaPopover.demo.vue:30]:
```suggestion
              placement="left"
              message="Placement Left"
              class="mr-4"
            >
              <va-button>Left</va-button>
```
and the same below

File: packages/ui/src/components/va-popover/VaPopover.vue [hunk 5]
```diff
@@ -47,12 +39,14 @@
...
 import { computed, defineComponent, PropType } from 'vue'
-import VaDropdown from '../va-dropdown/VaDropdown.vue'
-import VaIcon from '../va-icon/VaIcon.vue'
+
 import { extractComponentProps, filterComponentProps } from '../../utils/child-props'
 import { useColors } from '../../composables/useColor'
 import { useTextColor } from '../../composables/useTextColor'
 
+import VaDropdown from '../va-dropdown/VaDropdown.vue'
+import VaIcon from '../va-icon/VaIcon.vue'
```
Comment (RVitaly1978) [inline packages/ui/src/components/va-popover/VaPopover.vue:48]:
```suggestion
import { VaDropdown } from '../va-dropdown'
import { VaIcon } from '../va-icon'
```

File: packages/ui/src/components/va-popover/_variables.scss [hunk 6]
```diff
@@ -6,8 +6,8 @@
   --va-popover-content-padding: 0.65rem 1rem;
   --va-popover-content-border-radius: 0.5rem;
   --va-popover-content-font-size: 1rem;
-  --va-popover-content-wrapper-background-color: #ffffff;
-  --va-popover-content-wrapper-border-radius: 0.5rem;
+  --va-popover-content-background-color: #ffffff;
   --va-popover-title-font-weight: 700;
   --va-popover-title-margin-bottom: 0.125rem;
+  --va-popover-body-line-height: 1.5rem;
```
Comment (RVitaly1978) [inline packages/ui/src/components/va-popover/_variables.scss:12]:
```suggestion
  --va-popover-body-line-height: 1.5;
```

File: packages/ui/src/components/va-popover/_variables.scss [hunk 7]
```diff
@@ -6,8 +6,8 @@
   --va-popover-content-padding: 0.65rem 1rem;
   --va-popover-content-border-radius: 0.5rem;
   --va-popover-content-font-size: 1rem;
-  --va-popover-content-wrapper-background-color: #ffffff;
-  --va-popover-content-wrapper-border-radius: 0.5rem;
+  --va-popover-content-background-color: #ffffff;
```
Comment (RVitaly1978) [inline packages/ui/src/components/va-popover/_variables.scss:9]:
```suggestion
  --va-popover-content-background-color: var(--va-white);
```

**Review by RVitaly1978 — CHANGES_REQUESTED** [review]

**Review by RVitaly1978 — APPROVED** [review]

**Pull request merged**